tetrateci/tsb-util/k8s-objects/traffic-gen.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: trafficgenerator
+  name: trafficgenerator
+  namespace: bookinfo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: trafficgenerator
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+      labels:
+        app: trafficgenerator
+    spec:
+      containers:
+      - args:
+        - /bin/sh
+        - -c
+        - |
+          while :; do
+            curl "https://bookinfo.tetrate.com/productpage" --resolve "bookinfo.tetrate.com:443:34.93.184.210" --cacert /etc/bookinfo/bookinfo-ca.crt #>/dev/null 2>&1
```

Comment:
ip address has to be templatized. Take it from env and make the env var has a template var.

Reply:
I am not utilizing this yaml at all, how should I be doing that?